### DISCUSSION TITLE
server.crt blank

### DISCUSSION THREAD

**Discussion by miguelsmuller** [epoptes/epoptes#96] [Q&A]
I have updated my ubuntu and my epoptes now I can no longer do the 'epoptes-client' -c look at the file server.crt in /etc/epoptes and the file is empty.

**Comment by alkisg** [comment]
This probably means that at some point, you ran `epoptes-client -c` on the server itself, so it tried to write the file while fetching, and ended up zeroing it.
You need to generate a new certificate. One way to do that is `sudo apt purge epoptes; sudo apt install epoptes`.

**Comment by miguelsmuller** [comment]
ok ... it worked perfectly here ...
I managed to run `epoptes-client -c` on clients perfectly now ...

thank you very much.
